### DISCUSSION TITLE
buddhism/2023/maranasati/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#63] [General]
# buddhism/2023/maranasati/

https://shanenull.com/buddhism/2023/maranasati/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2021/confusion-to-clarity/

**Reply by shane0** [reply]
https://en.wikipedia.org/wiki/Mara%E1%B9%87asati

**Reply by shane0** [reply]
https://youtu.be/P4Nog-3hNvo

**Reply by shane0** [reply]
https://youtu.be/P4Nog-3hNvo

**Reply by shane0** [reply]
https://youtu.be/wL5IjuPAEfU

**Reply by shane0** [reply]
https://youtu.be/wL5IjuPAEfU

**Reply by shane0** [reply]
- longer video https://youtu.be/V4j5RJfu5X0

**Comment by shane0** [comment]
https://shanenull.com/stoicism/memento_mori/#challenge-who-are-you-at-your-own-funeral

**Reply by shane0** [reply]
5 recollections https://www.dhammatalks.org/suttas/AN/AN5_57.html

**Reply by shane0** [reply]
https://youtu.be/aaNq1DenFh8?si=v2pNHD63SIlutOvh

**Comment by shane0** [comment]
Art of dying
 Die before you die
So you do not die when you die

https://youtube.com/clip/UgkxSFspsgqsmorpNkgxLFfuySPn0A0HiDVs?si=hR5PtFkOo23OteLH

**Comment by shane0** [comment]
https://www.dhammatalks.org/books/ChantingGuide/Section0021.html
https://puredhamma.net/key-dhamma-concepts/basic-framework-of-buddha-dhamma/tilakkhana-introduction/how-does-anicca-nature-lead-to-dukkha/

**Comment by shane0** [comment]
- moved to 2024 https://shanenull.com/buddhism/2024/maranasati/
